windows-driver-docs-pr/image/sample-inf-file-for-a-web-services-scanner.md

```diff
@@ -92,11 +86,9 @@ MyWIADriver.dll
 
 [WSDScanDriver.AddReg]
 HKR,,HardwareConfig,1,1 ; generic WDM device
-HKR,,USDClass,,"{00000000-0000-0000-0000-000000000000}"
-; the GUID for the WIA mini-driver
-HKCR,CLSID\{00000000-0000-0000-0000-000000000000},,,"<Description
-of your Web Services scanner WIA device>"
-HKCR,CLSID\{00000000-0000-0000-0000-000000000000}\InProcServer32,,,%11%\YourWIADriver.dll
+HKR,,USDClass,,"{00000000-0000-0000-0000-000000000000}"; the GUID for the WIA mini-driver
+HKCR,CLSID\{00000000-0000-0000-0000-000000000000},,,"<Description of your Web Services scanner WIA device>"
+HKCR,CLSID\{00000000-0000-0000-0000-000000000000}\InProcServer32,,,%%SystemRoot%%\YourWIADriver.dll
```

Comment:
Your change to the DefaultDestDir to make it 13 ('run from driverstore') means that the file won't be at %systemroot%\YourWIADriver.dll.  You would need to use %13%\YourWIADriver.dll here.